pkg/test/resourcefixture/testdata/basic/bigqueryconnection/v1alpha1/bigqueryconnectionconnection/cloudsqlconnectionbasic/_http.log

```diff
@@ -778,7 +778,7 @@ X-Xss-Protection: 0
       "instance": "sqlinstance-sample-${uniqueId}",
       "kind": "sql#user",
       "name": "sqluser-${uniqueId}",
-      "password": "password",
+      "password": "cGFzc3dvcmQ=",
```

Comment:
Uhoh ... should the password be provided in plaintext or base64?

Edit: nevermind, I see why, but I suggest using a password that doesn't cause confusion (like "password" or "secret")